### PR TITLE
February Update

### PR DESCRIPTION
**Pull request by garrying**
- [x] Updated `title` attribute (_State of Our Networks_ ➡️ _Our Networks_) (5abf528)
- [x] Remove Cotham Sans and Terminal Grotesque
- [x] Update `meta` image (b/w logo)
- [x] Revise accent colour (splash, and links)
- [x] Stretch: revise mobile menu layout
- [x] Stretch: add Save the Date and 2019 Theme (#92)
  - [ ] add email scroll
  - [ ]  animate backdrop?
- [ ] Stretch: add Meetup events to homepage

### CODE REVIEW

**Review by ana0 — APPROVED** [review]

**Comment by benhylau** [comment]
I tried subscribing to updates and got this:

screen shot 2019-02-28 at 11 19 59 pm

Both on prod site and this PR. Thoughts?

File: index.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ redirect_from:
...
+Our Networks 2019 explores the collective care and maintenance of alternative networking practices&mdash;new protocols, peer-to-peer connections, offline-first computing, or community-based governance and ownership. Building on last year, where we asked how to move beyond solitary practices of do it yourself (DIY), in 2019 we explore what critical and creative practices offer **RE: Infrastructures** of alternatives.
+
+How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine radically different futures, resist easy co-option by the forces of late-stage capitalism, and work in solidarity as plural networks seeking emancipatory change? To answer this we draw on insights from organizers, technologists, artists, and scholars to ensure the focus is not only on the technical aspects of infrastructure, but also the social and cultural. Their diverse perspectives suggest ways we can make space for livelihoods, solidarity, and survival; draw on history and place to bring our fullest imaginations to this moment; and open ways to think these questions together as we build transitional forms.
```
Comment (benhylau) [inline index.md]: Can we restrict the flow that results from the typing effect to a single line like this? I tried to read the paragraph and got dizzy 😭 

```suggestion
How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine  
radically different futures, resist easy co-option by the forces of late-stage capitalism, and work in solidarity as plural networks seeking emancipatory change? To answer this we draw on insights from organizers, technologists, artists, and scholars to ensure the focus is not only on the technical aspects of infrastructure, but also the social and cultural. Their diverse perspectives suggest ways we can make space for livelihoods, solidarity, and survival; draw on history and place to bring our fullest imaginations to this moment; and open ways to think these questions together as we build transitional forms.
```

File: _sass/_typography.scss [hunk 2]
```diff
@@ -109,15 +61,63 @@ p {
...
+  color: rgba($text-color, 0.8);
+  font-family: $mono-font-family;
+  font-size: $small-font-size;
+  line-height: 1.25em;
+  padding: 1em;
+  margin: 2em 0;
+  width: 75%;
+  background: rgba(0,0,0,.05);
+  height: 18em;
+  overflow: scroll;
```
Comment (benhylau) [inline _sass/_typography.scss]: This gets rid of the x scrollbar and shows the y scrollbar when needed:

```suggestion
  overflow: auto;
```

**Review by benhylau — APPROVED** [review]
Please see my two suggested edits. Otherwise this looks great!

**Comment by garrying** [comment]
Note: The issue ticket around mailman subscribe form on mayfirst https://support.mayfirst.org/ticket/14497

File: index.md (hunk 1 above)
Comment (dcwalk) [reply index.md]: This change causes some v. strange wrap issues on mobile and non. I think we should revisit the decision to add in a line break. Maybe adjust the animation speed? @garrying ?

**Comment by dcwalk** [comment]
I think we might want to consider two minor updates before a major overhaul:
- revisit how the linebreak is handled (I'm getting odd wrap issues which look worse to me than the scrolling text, mabe we just slow down the animation :woman_shrugging: 
- think about styling the scrollbar? or something? I think it looks a bit... constrained in a way that breaks the aesthetic as it is currently formed.

**Pull request merged**